projector-plugin/src/main/kotlin/ProjectorService.kt

```diff
@@ -62,24 +62,19 @@ class ProjectorService : PersistentStateComponent<ProjectorConfig> {
   private val logger = Logger.getInstance(ProjectorService::class.java)
   private val plugin = PluginManager.getPlugin(PluginId.getId("org.jetbrains.projector-plugin"))!!
 
-  val host: String?
-    get() = config.host
-
-  val port: String?
-    get() = config.port
-
-  var currentSession: Session? = null
+  private var currentSession: Session? = null
     set(value) {
       field = value
       config.host = value?.host
       config.port = value?.port
     }
 
-  var enabled: EnabledState = when (areRequiredVmOptionsPresented()) {
+  private var enabled: EnabledState = when (areRequiredVmOptionsPresented()) {
     true -> EnabledState.HAS_VM_OPTIONS_AND_DISABLED
     false -> EnabledState.NO_VM_OPTIONS_AND_DISABLED
   }
-    private set
+    private set // TODO: fix. Not redundant - get() used only once at initialization time. Next invocations of get() uses field, which is
+                // modified when session started. Without setter menu will stuck in inital state "Start Remote Access...".
```

Comment:
```suggestion
```

I'm not sure why we can't just remove the setter... I even removed it locally and everything seemed to continue working fine.

Am I right that `private set` generates the same setter as default one generated for `private var`?